samples/matter/bridge/src/bridged_device_types/humidity_sensor.h

```diff
@@ -26,12 +29,15 @@ class HumiditySensorDevice : public BridgedDevice {
 	{
 		return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
 	}
+	CHIP_ERROR HandleAttributeChange(chip::ClusterId clusterId, chip::AttributeId attributeId, void *data,
+					 size_t dataSize) override;
 
 private:
-	static constexpr uint16_t kRelativeHumidityMeasurementClusterRevision = 1;
-	static constexpr uint32_t kRelativeHumidityMeasurementFeatureMap = 0;
+	void SetMeasuredValue(int16_t value) { mMeasuredValue = value; }
+	void SetMinMeasuredValue(int16_t value) { mMinMeasuredValue = value; }
+	void SetMaxMeasuredValue(int16_t value) { mMaxMeasuredValue = value; }
 
-	uint16_t mMeasuredValue = 50;
-	uint16_t mMinMeasuredValue = 0;
-	uint16_t mMaxMeasuredValue = 100;
+	uint16_t mMeasuredValue = 0;
+	uint16_t mMinMeasuredValue = CONFIG_BRIDGE_HUMIDITY_SENSOR_MIN_MEASURED_VALUE;
+	uint16_t mMaxMeasuredValue = CONFIG_BRIDGE_HUMIDITY_SENSOR_MAX_MEASURED_VALUE;
```

Comment:
Why do we want to set these parameters runtime?

Reply:
I wasn't sure actually. The Matter spec doesn't allow to modify these values by the external devices (e.g. another Matter device), however it can be modified locally by the device. I was wondering whether it's a valid use case that Bluetooth LE device would want to modify the measured range at runtime and thus modify the clusters attributes as well.